atlp-rwanda/ecommerce-app-predators-bn
# feature(Authorization):Roles and Permissions

#### What does this PR do?

Create Roles and Set Permissions 

#### Description of Task to be completed?
Admin should be able to assign various users to various roles. There should also be a role for the admin account itself.
This will allow only accounts with particular roles permission to peform particular actions.

#### How should this be manually tested?

Checkout this branch.
Run npm install to install all dependencies.
Run npm run dev
Check the console output for test results.
### Endpoints
Work in progress

#### Any background context you want to provide?
To ensure security and prevent unauthorized access, it is important to set specific permissions for each role and
restrict access to sensitive data and features. For example, only the admin should be able to access and modify 
site settings and vendor information, while vendors should only be able to access and modify their own products and orders.
Regular audits and updates to permissions can help ensure that the platform remains secure and functional for all users.

#### What are the relevant pivotal tracker/Trello stories?
https://trello.com/c/05jrYDhh

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
